templates/helpers.js

```diff
@@ -240,8 +241,7 @@ module.exports = function(docMap, options, getCurrent, helpers, OtherHandlebars)
                 return false;
             }
             var name = packageObject.name,
```

Comment:
It looks like it's the `docObject` or `docMap` in it's entirety.

https://github.com/bit-docs/bit-docs-tag-package/blob/master/package.js

https://github.com/canjs/bit-docs-html-canjs/blob/master/templates/title.mustache#L32